apps/backend/src/events/entities/event.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { IsDate, IsNotEmpty, IsString, IsUUID } from 'class-validator';
+
+export class Event {
+  @IsUUID()
+  id: string;
+
+  @IsString()
+  @IsNotEmpty()
+  name: string;
+
+  @IsString()
+  @IsNotEmpty()
+  location: string;
+
+  @IsDate()
+  startDate: Date;
+
+  @IsDate()
+  endDate: Date;
```

Comment:
we will probably need @IsDateString() here, IsDate expects a Date object, but wie'll get a string

Reply:
also, it'd be cool to validate that endDate is after startDate. You probably have to create a custom decorator for that, it's a great task if you have the time